From: Xueshi Hu <xueshi.hu@smartx.com>
Subject: mm: remove clear_page_idle()
Date: Wed, 12 Jul 2023 21:49:59 +0800

All callers have now been converted to call folio_clear_idle().

Link: https://lkml.kernel.org/r/20230712134959.145373-1-xueshi.hu@smartx.com
Signed-off-by: Xueshi Hu <xueshi.hu@smartx.com>
Reviewed-by: David Hildenbrand <david@redhat.com>
Cc: Charan Teja Kalla <quic_charante@quicinc.com>
Cc: Michal Hocko <mhocko@suse.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/page_idle.h |    5 -----
 1 file changed, 5 deletions(-)

--- a/include/linux/page_idle.h~mm-remove-clear_page_idle
+++ a/include/linux/page_idle.h
@@ -144,9 +144,4 @@ static inline void set_page_idle(struct
 {
 	folio_set_idle(page_folio(page));
 }
-
-static inline void clear_page_idle(struct page *page)
-{
-	folio_clear_idle(page_folio(page));
-}
 #endif /* _LINUX_MM_PAGE_IDLE_H */
_
